configs/config.go

```diff
@@ -1,36 +1,36 @@
-package main
+package configs
```

Comment:
Can we make this just `config` singular? It feels awkward having this as a plural. Partly I think I would lean toward package names in general being singular.

Reply:
Yup, agree with you that sounds weird. Fixed!